### ISSUE TITLE
Incorrect HTML rendering in Markdown

### ISSUE THREAD

**Issue by jurgob** [sapegin/react-styleguidist#188]
I'm using jsdoc2md to include some md file in my styleguide docs, but I see some bad rendering (look the image)
![screen shot 2016-09-23 at 11 02 29](https://cloud.githubusercontent.com/assets/6890893/18782217/415bb76a-817d-11e6-9cd9-067fe4794b5e.png)

**Comment by sapegin** [comment]
Markdown source would help debug the issue.

**Comment by jurgob** [comment]
```js

## Modules

<dl>
<dt><a href="#module_actions/auth">actions/auth</a></dt>
<dd></dd>
<dt><a href="#module_actions/cron">actions/cron</a></dt>
<dd></dd>
<dt><a href="#module_actions/geo">actions/geo</a></dt>
<dd></dd>
<dt><a href="#module_actions/globals">actions/globals</a></dt>
<dd></dd>
<dt><a href="#module_actions/monitor">actions/monitor</a></dt>
<dd></dd>
<dt><a href="#module_actions/profiles">actions/profiles</a></dt>
<dd></dd>
<dt><a href="#module_actions/proxy">actions/proxy</a></dt>
<dd></dd>
<dt><a href="#module_actions/search">actions/search</a></dt>
<dd></dd>
<dt><a href="#module_actions/socialgraph">actions/socialgraph</a></dt>
<dd></dd>
</dl>

<a name="module_actions/auth"></a>

## actions/auth

* [actions/auth](#module_actions/auth)
    * [.addUser](#module_actions/auth.addUser)
    * [.modifyUser](#module_actions/auth.modifyUser)
    * [.userAddToken](#module_actions/auth.userAddToken)
    * [.setUsersGroups](#module_actions/auth.setUsersGroups)
    * [.addUsersGroups](#module_actions/auth.addUsersGroups)

<a name="module_actions/auth.addUser"></a>

### actions/auth.addUser
**Kind**: static constant of <code>[actions/auth](#module_actions/auth)</code>  

| Param | Type | Description |
| --- | --- | --- |
| userId | <code>string</code> | user it to add |

<a name="module_actions/auth.modifyUser"></a>

### actions/auth.modifyUser
**Kind**: static constant of <code>[actions/auth](#module_actions/auth)</code>  

| Param | Type | Description |
| --- | --- | --- |
| params | <code>object</code> | user object |
| userId | <code>string</code> | user it to modify |

<a name="module_actions/auth.userAddToken"></a>

### actions/auth.userAddToken
**Kind**: static constant of <code>[actions/auth](#module_actions/auth)</code>  

| Param | Type | Description |
| --- | --- | --- |
| userId | <code>string</code> | user id |

<a name="module_actions/auth.setUsersGroups"></a>

### actions/auth.setUsersGroups
**Kind**: static constant of <code>[actions/auth](#module_actions/auth)</code>  

| Param | Type | Description |
| --- | --- | --- |
| groups | <code>array</code> | groups list |
| groups[].gid | <code>string</code> | group gid |
| groups[].name | <code>string</code> | group name |

<a name="module_actions/auth.addUsersGroups"></a>

### actions/auth.addUsersGroups
**Kind**: static constant of <code>[actions/auth](#module_actions/auth)</code>  

| Param | Type | Description |
| --- | --- | --- |
| data | <code>object</code> | data |
| data.gid | <code>string</code> | gid |
| data.name | <code>string</code> | gid |

**Example**  
```js
addUsersGroups({gid:"one", name:"my name"})
```
<a name="module_actions/cron"></a>

## actions/cron

* [actions/cron](#module_actions/cron)
    * [.jobListReceved](#module_actions/cron.jobListReceved)
    * [.editJobStart](#module_actions/cron.editJobStart)
    * [.deleteJob](#module_actions/cron.deleteJob)
    * [.editJobRequest](#module_actions/cron.editJobRequest)

<a name="module_actions/cron.jobListReceved"></a>

### actions/cron.jobListReceved
**Kind**: static constant of <code>[actions/cron](#module_actions/cron)</code>  

| Param | Type | Description |
| --- | --- | --- |
| jobs | <code>array</code> | list of job |

<a name="module_actions/cron.editJobStart"></a>

### actions/cron.editJobStart
**Kind**: static constant of <code>[actions/cron](#module_actions/cron)</code>  

| Param | Type | Description |
| --- | --- | --- |
| job_id | <code>string</code> | job id to modify |

<a name="module_actions/cron.deleteJob"></a>

### actions/cron.deleteJob
**Kind**: static constant of <code>[actions/cron](#module_actions/cron)</code>  

| Param | Type | Description |
| --- | --- | --- |
| job_id | <code>string</code> | job id to delete |

<a name="module_actions/cron.editJobRequest"></a>

### actions/cron.editJobRequest
**Kind**: static constant of <code>[actions/cron](#module_actions/cron)</code>  

| Param | Type | Description |
| --- | --- | --- |
| job_id | <code>string</code> | job id |
| data | <code>params</code> | new job data |

<a name="module_actions/geo"></a>

## actions/geo
<a name="module_actions/globals"></a>

## actions/globals

* [actions/globals](#module_actions/globals)
    * [.successMessage](#module_actions/globals.successMessage)
    * [.errorMessage](#module_actions/globals.errorMessage)
    * [.setExpandMenu](#module_actions/globals.setExpandMenu)

<a name="module_actions/globals.successMessage"></a>

### actions/globals.successMessage
**Kind**: static constant of <code>[actions/globals](#module_actions/globals)</code>  

| Param | Type | Description |
| --- | --- | --- |
| successMessage | <code>string</code> | set a message, if is equals to "" message panel  will be hidden |

<a name="module_actions/globals.errorMessage"></a>

### actions/globals.errorMessage
**Kind**: static constant of <code>[actions/globals](#module_actions/globals)</code>  

| Param | Type | Description |
| --- | --- | --- |
| errorMessage | <code>string</code> | set error message, if is equals to "" message panel  will be hidden |

<a name="module_actions/globals.setExpandMenu"></a>

### actions/globals.setExpandMenu
**Kind**: static constant of <code>[actions/globals](#module_actions/globals)</code>  

| Param | Type | Description |
| --- | --- | --- |
| expanded | <code>bool</code> | expand side menu, true/false |

<a name="module_actions/monitor"></a>

## actions/monitor

* [actions/monitor](#module_actions/monitor)
    * [.monitorExtractorSetProperty](#module_actions/monitor.monitorExtractorSetProperty)
    * [.monitorSgnSetProperty](#module_actions/monitor.monitorSgnSetProperty)
    * [.monitorProxyStatsProxyReceved](#module_actions/monitor.monitorProxyStatsProxyReceved)
    * [.monitorProxyStatsProviderReceved](#module_actions/monitor.monitorProxyStatsProviderReceved)
    * [.monitorProxyStatsUrlReceved](#module_actions/monitor.monitorProxyStatsUrlReceved)
    * [.monitorProxyUrlReceved](#module_actions/monitor.monitorProxyUrlReceved)
    * [.monitorProxyUrlDelete](#module_actions/monitor.monitorProxyUrlDelete)
    * [.monitorProxyUrlPost](#module_actions/monitor.monitorProxyUrlPost)
    * [.monitorProxyUrlStatusPut](#module_actions/monitor.monitorProxyUrlStatusPut)

<a name="module_actions/monitor.monitorExtractorSetProperty"></a>

### actions/monitor.monitorExtractorSetProperty
**Kind**: static constant of <code>[actions/monitor](#module_actions/monitor)</code>  

| Param | Type | Description |
| --- | --- | --- |
| source | <code>string</code> | type of source to be showed |
| prop | <code>string</code> | method to be triggered, it can be: run \ reset \ auto \ working |

<a name="module_actions/monitor.monitorSgnSetProperty"></a>

### actions/monitor.monitorSgnSetProperty
**Kind**: static constant of <code>[actions/monitor](#module_actions/monitor)</code>  

| Param | Type | Description |
| --- | --- | --- |
| source | <code>string</code> | type of source to be showed |
| prop | <code>string</code> | method to be triggered, it can be: run \ reset \ auto \ working |

<a name="module_actions/monitor.monitorProxyStatsProxyReceved"></a>

### actions/monitor.monitorProxyStatsProxyReceved
**Kind**: static constant of <code>[actions/monitor](#module_actions/monitor)</code>  

| Param | Type | Description |
| --- | --- | --- |
| proxy_stats | <code>object</code> | proxy props |

<a name="module_actions/monitor.monitorProxyStatsProviderReceved"></a>

### actions/monitor.monitorProxyStatsProviderReceved
**Kind**: static constant of <code>[actions/monitor](#module_actions/monitor)</code>  

| Param | Type | Description |
| --- | --- | --- |
| provider_stats | <code>object</code> | providers props |

<a name="module_actions/monitor.monitorProxyStatsUrlReceved"></a>

### actions/monitor.monitorProxyStatsUrlReceved
**Kind**: static constant of <code>[actions/monitor](#module_actions/monitor)</code>  

| Param | Type | Description |
| --- | --- | --- |
| url_stats | <code>object</code> | url stats |

<a name="module_actions/monitor.monitorProxyUrlReceved"></a>

### actions/monitor.monitorProxyUrlReceved
**Kind**: static constant of <code>[actions/monitor](#module_actions/monitor)</code>  

| Param | Type | Description |
| --- | --- | --- |
| proxy_url | <code>object</code> | url proxy monitored |

<a name="module_actions/monitor.monitorProxyUrlDelete"></a>

### actions/monitor.monitorProxyUrlDelete
**Kind**: static constant of <code>[actions/monitor](#module_actions/monitor)</code>  

| Param | Type | Description |
| --- | --- | --- |
| url | <code>string</code> | url to remove from monitoring |

<a name="module_actions/monitor.monitorProxyUrlPost"></a>

### actions/monitor.monitorProxyUrlPost
**Kind**: static constant of <code>[actions/monitor](#module_actions/monitor)</code>  

| Param | Type | Description |
| --- | --- | --- |
| url | <code>string</code> | url to add from monitoring |

<a name="module_actions/monitor.monitorProxyUrlStatusPut"></a>

### actions/monitor.monitorProxyUrlStatusPut
**Kind**: static constant of <code>[actions/monitor](#module_actions/monitor)</code>  

| Param | Type | Description |
| --- | --- | --- |
| url | <code>string</code> | url to add from monitoring |
| status | <code>bool</code> | url to add from monitoring |

<a name="module_actions/profiles"></a>

## actions/profiles
<a name="module_actions/proxy"></a>

## actions/proxy
<a name="module_actions/search"></a>

## actions/search
<a name="module_actions/socialgraph"></a>

## actions/socialgraph

* [actions/socialgraph](#module_actions/socialgraph)
    * [.socialGraphRequest](#module_actions/socialgraph.socialGraphRequest)
    * [.socialGraphReceved](#module_actions/socialgraph.socialGraphReceved)
    * [.socialGraphSelectNode](#module_actions/socialgraph.socialGraphSelectNode)
    * [.socialGraphDeleteSgn](#module_actions/socialgraph.socialGraphDeleteSgn)

<a name="module_actions/socialgraph.socialGraphRequest"></a>

### actions/socialgraph.socialGraphRequest
**Kind**: static constant of <code>[actions/socialgraph](#module_actions/socialgraph)</code>  

| Param | Type | Description |
| --- | --- | --- |
| root | <code>array</code> | list of roots cdn links, e.g.: ["sgn://youtube.com/?ident=vaneeesab", "sgn://twitter.com/?ident=ppolleunus"] |

<a name="module_actions/socialgraph.socialGraphReceved"></a>

### actions/socialgraph.socialGraphReceved
**Kind**: static constant of <code>[actions/socialgraph](#module_actions/socialgraph)</code>  

| Param | Type | Description |
| --- | --- | --- |
| socialgraph | <code>object</code> | socilal graph object |

<a name="module_actions/socialgraph.socialGraphSelectNode"></a>

### actions/socialgraph.socialGraphSelectNode
**Kind**: static constant of <code>[actions/socialgraph](#module_actions/socialgraph)</code>  

| Param | Type | Description |
| --- | --- | --- |
| node_sgn | <code>string</code> | sgn string |

<a name="module_actions/socialgraph.socialGraphDeleteSgn"></a>

### actions/socialgraph.socialGraphDeleteSgn
**Kind**: static constant of <code>[actions/socialgraph](#module_actions/socialgraph)</code>  

| Param | Type | Description |
| --- | --- | --- |
| sgn | <code>string</code> | sgn string |



```

**Comment by jurgob** [comment]
here you have a subset of it

**Comment by sapegin** [comment]
Thanks! Looks like some problems with HTML handling.

**Comment by sapegin** [comment]
Should be fixed in [4.0.0](https://github.com/sapegin/react-styleguidist/releases/tag/4.0.0).
